db/schema.sql

```diff
@@ -0,0 +1,4 @@
+DROP DATABASE IF EXISTS theDB;
```

Comment:
Let's call the Database 'TutorificDB'
Tables being the names you've already defined.